els-back/app/Http/Controllers/User/UserController.php

```diff
@@ -15,15 +16,26 @@ public function index(Request $request)
         return User::paginate($per_page);
     }
 
+    public function show(Request $request, User $user)  
+    {
+        $authenticated_user = $request->user();
+        $user->load('learned_words.word','follows','followers','results');
+    
+        $user->activities = Activity::whereIn('user_id',$user->scopeMineAndFollowingIds())->with('activityable','user')->orderBy('created_at','desc')->get();
```

Comment:
I believe we can remove the scope when calling it.
```suggestion
        $user->activities = Activity::whereIn('user_id',$user->mineAndFollowingIds())->with('activityable','user')->orderBy('created_at','desc')->get();
```